lib/FunctionIsolation/IsolateFunctions.cpp

```diff
@@ -183,14 +184,15 @@ class IsolateFunctionsImpl {
     Context(TheModule->getContext()),
     GCBI(GCBI),
     Binary(Binary),
+    NamingHelper(Binary),
```

Comment:
Does this mean we're constructing the global namespace just to then call `LLVMFunction` which, unless we have `DebugNames` turned on, doesn't use it?

Reply:
Yep, and it pains me to do that. I guess we should sneak one more `DebugNames` check which is also sad.

Or, maybe, provide another way to make `NamingHelper` (template argument) which ignores global namespace. We don't care about trailing `_`s as much in debug names.

Reply:
Closing this as it kind of duplicates the next thread.